### PR TITLE
SWS-334 Add missing Adapter/Template objects in the backend

### PR DESCRIPTION
**Pull request by lucasponce**
- Adapters: Circonus, Fluentd, Kubernetes Env, Memory quota, OPA, Prometheus, RBAC, Redis Quota, Service Control, Solar Winds, StatsD, Stdio
- Templates: API Key, Authorization, Kubernetes, Log Entry, Metric, Quota, Report Nothing, Service Control Report

### CODE REVIEW

File: kubernetes/istio_rules_service.go [hunk 1]
```diff
@@ -111,10 +111,15 @@ func (in *IstioClient) getActionHandler(namespace string, handlerType string, ha
 		return
 	}
 	result, err := in.istio.Get().Namespace(namespace).Resource(handlerTypePlural).SubResource(handlerName).Do().Get()
+	log.Infof("TODELETE %s/%s/%s", handlerTypePlural, handlerType, handlerName)
+	log.Infof("TODELETE %v", result)
+	log.Infof("TODELETE %v", err)
```
Comment (xeviknal) [inline kubernetes/istio_rules_service.go]: I guess those logs are not intended be here, right?

Comment (lucasponce) [reply kubernetes/istio_rules_service.go]: yeah, I completely forgot to clean them.
Good catch

**Comment by lucasponce** [comment]
These files are created from a template using some helper scripts attached into the JIRA.
https://issues.jboss.org/browse/SWS-334
In theory, this is a one time operation, so it is not expected to change them, that's why I didn't add these scripts into the hack folder.

Note, this is a similar way as Istio Pilot manages these models.

**Comment by pilhuhn** [comment]
Is there no way of importing them as a dependency instead of putting the files in here?

**Comment by lucasponce** [comment]
> Is there no way of importing them as a dependency instead of putting the files in here?

Not in a easy step.
These classes are not all available from Istio in a single dependency we can use, we finish pulling undesired dependencies.

Istio uses a strategy to put all in a "generated" file called types, if people are not against that, it makes sense.

In the mid term, another refactoring strategy is to not use apimachinery dependency from k8s and check a different approach for marshalling.

But the tradeof was not positive, so for this step I guess we can add them, or put them in a single auto-generate file (as istio does).

**Pull request merged**